src/Orleans/Streams/PubSub/ImplicitStreamSubscriberTable.cs

```diff
@@ -230,38 +210,23 @@ private IStreamConsumerExtension MakeConsumerReference(IInternalGrainFactory gra
         }
 
         /// <summary>
-        /// Collects the namespaces associated with a grain class type through the use of ImplicitStreamSubscriptionAttribute.
+        /// Collects the namespace predicates associated with a grain class type through the use of
+        /// <see cref="ImplicitStreamSubscriptionAttribute"/>.
         /// </summary>
-        /// <param name="grainClass">A grain class type that might have ImplicitStreamSubscriptionAttributes associated with it.</param>
-        /// <returns></returns>
-        /// <exception cref="System.ArgumentException">grainType does not describe a grain class.</exception>
-        /// <exception cref="System.InvalidOperationException">duplicate specification of ImplicitConsumerActivationAttribute(...).</exception>
-        private static ISet<string> GetNamespacesFromAttributes(Type grainClass)
+        /// <param name="grainClass">A grain class type that might have
+        /// attributes of type <see cref="ImplicitStreamSubscriptionAttribute"/>  associated with it.</param>
+        /// <returns>The list of stream namespace predicates.</returns>
+        /// <exception cref="System.ArgumentException"><paramref name="grainClass"/> does not describe a grain class.</exception>
+        private static IList<IStreamNamespacePredicate> GetPredicatesFromAttributes(Type grainClass)
         {
             if (!TypeUtils.IsGrainClass(grainClass))
             {
-                throw new ArgumentException(string.Format("{0} is not a grain class.", grainClass.FullName), "grainClass");
+                throw new ArgumentException($"{grainClass.FullName} is not a grain class.", nameof(grainClass));
             }
 
-            var attribs = grainClass.GetTypeInfo().GetCustomAttributes<ImplicitStreamSubscriptionAttribute>(inherit: false);
+            var attribs = grainClass.GetTypeInfo().GetCustomAttributes<ImplicitStreamSubscriptionAttribute>(false);
```

Comment:
I think you are correct. I misread the documentation as 
```
inherit
Type: System.Boolean
true to inspect the descendants of element; otherwise, false. 
```
The correct doc is `true to inspect the ancestors of element; otherwise, false. `